design/ceph/object/ceph-object-realm.md

```diff
@@ -0,0 +1,73 @@
+# Rook Ceph Object Realm
+
+## Prerequisites
+
+A Rook Ceph cluster.
+
+## Ceph Object Realm Walkthrough
+
+The resource descibed in this design document represents the realm in the [Ceph Multisite data model](/design/ceph/object/ceph-multisite-overview.md).
+
+### Creating an Object Realm
+
+#### Config
+
+When the storage admin is ready to create a multisite realm for object storage, the admin will name the realm in the metadata section on the configuration file.
+
+This example `ceph-object-realm.yaml`, names a realm `my-realm`.
+```yaml
+apiVersion: ceph.rook.io/v1alpha1
+kind: CephObjectRealm
+metadata:
+  name: my-realm
+  namespace: rook-ceph
+spec:
+```
+
+Now create the object realm.
+```bash
+kubectl create -f ceph-object-realm.yaml
+```
+
+#### Steps
+
+1. At this point the Rook operator recognizes that a new ceph-object-realm resource needs to be configured. The operator will start creating the ceph-object-realm resource.
+
+2. After these steps the admin should create:
+    - A [ceph-object-zone-group](/design/ceph/object/ceph-object-zone-group.md) referring to the ceph-object-realm resource.
+    - A [ceph-object-zone](/design/ceph/object/ceph-object-zone.md) referring to the ceph-object-zone-group resource.
+    - A [ceph-object-store](/design/ceph/object/ceph-object-store.md) referring to the ceph-object-zone resource.
+
+The order in which these resources are created is not important. 
+
+3. Once all of the resources from step #2 are started up, the operator will create a realm on the Rook Ceph cluster and the ceph-realm resource will be running.
+
+## Deleting and Reconfiguring the Object Realm
+
+At the moment creating an ceph-object-realm resource only handles Day 1 initial configuration for the realm. 
+
+Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+
+To be clear, when the ceph-object-realm resource is deleted or modified, the realm is not deleted from the Ceph cluster. Realm deletion must be done the toolbox.
+
+### Deleting a Realm
+
+The Rook toolbox can modify the Ceph Multisite state via the radosgw-admin command. 
+
+The following command, run via the toolbox, deletes the realm.
+
+`$ radosgw-admin realm rm --rgw-realm=my-realm`
+
+## Ceph-Object-Realm CRD
+
+The ceph-object-realm settings are exposed to Rook as a Custom Resource Definition (CRD). The CRD is the Kubernetes-native means by which the Rook operator can watch for new resources.
```

Comment:
nit: I'd just talk about the realm settings rather than `ceph-object-realm`, but it's fine either way if you prefer this.